sagemaker-geospatial/london-mapmatch-and-reverse-geocode/london-mapmatch-and-reverse-geocode.ipynb

```diff
@@ -0,0 +1,1289 @@
+{
```

Comment:
<p>Where does this data come from?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>The data is drawn by myself :) I used geojson.io to draw a common tourist path in London</p>